sdk/java/pulumi/src/main/java/com/pulumi/resources/StackReference.java

```diff
@@ -22,23 +22,14 @@
  */
 public class StackReference extends CustomResource {
 
-    /**
-     * The name of the referenced stack.
-     */
     @Export(name = "name", type = String.class)
     private Output<String> name;
 
-    /**
-     * The outputs of the referenced stack.
-     */
     @Export(name = "outputs", type = Map.class, parameters = {String.class, Object.class})
-    public Output<Map<String, Object>> outputs;
+    private Output<Map<String, Object>> outputs;
```

Comment:
Interesting, this may be a minor breaking change but hopefully we can get away with this for now.